### DISCUSSION TITLE
Speicherpfad für Messungen und Backup bei CCU Proxmox installation

### DISCUSSION THREAD

**Discussion by bandimuc** [OpenCCU/OpenCCU#2696] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen, ich ziehe gerade mein System auf NUC um. Dazu würde ich gerne meine alten Diagramm Daten über nehmen. Leider kann ich den richtigen Pfad hierfür nicht finden. Welchen Standard Pfad nimmt das System, wenn keine USB / SD Card vorhanden ist ??

Besten Dank im Voraus

**Comment by Baxxy13** [comment]
`/usr/local/sdcard`
Aber Vorsicht, alles was darin landet ist nicht Teil von RaspberryMatic - Backups. 
Bist du irgendwann gezwungen ein Restore per Backup durchzuführen ist alles darin, was nicht anderweitig gesichert wurde, verloren.

**Reply by bandimuc** [reply]
Hallo Baxxy13, ich habe mein alten *.rrd files in das /usr/local/sdcard/measurement kopiert, jedoch werde diese von der CCU nicht erkannt. Hast Du eine Idee warum ?
Die Backups löse ich über Proxmox :)
Danke und Gruß

**Reply by jens-maus** [reply]
> Die Backups löse ich über Proxmox :)

Darauf solltest du dich nicht zu 100% verlassen. Immer besser zusätzlich noch täglich die sbk-Backups an einen sicheren Ort kopieren zu lassen.

**Comment by Baxxy13** [comment]
Ich nutze die interne Diagrammfunktion nicht.
Ist mir zu eingeschränkt und mit dem CCU-Historian gibt es eine weit bessere Alternative.

Nur die .rrd's zu kopieren wird nicht reichen.
Es braucht auch die Konfigurationsdateien (*.dic) die m.W. in `/usr/local/etc/config/measurement` liegen.
Wenn dein "Umzug" mittels Backup-Restore stattfand sollten die aber eigentlich vorhanden sein.

**Comment by bandimuc** [comment, marked as answer]
Hallo Baxxy13,Ich hatte das so zurück gespielt, hat aber nicht funktioniert.Ich werde nun auch auf Influx / Grafana umsteigen.Danke für die Unterstützung :)
